src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/CCJContentProvider.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.cmc.claimstore.services.staff.content;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.claimstore.models.Claim;
+
+import java.util.Collections;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Collections'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AUselessImportCheck)